README.md

```diff
@@ -1165,6 +1165,8 @@ See also [Are we game yet?](https://arewegameyet.rs)
   * [oxidator](https://github.com/Ruddle/oxidator) — A real time strategy game/engine written with Rust and WebGPU
   * [Piston](https://www.piston.rs/) — [![Crates.io](https://img.shields.io/crates/v/piston.svg?style=flat-square)](https://crates.io/crates/piston) [![Crates.io](https://img.shields.io/crates/l/piston.svg)](https://github.com/PistonDevelopers/piston/blob/master/LICENSE)
   * [Unrust](https://github.com/unrust/unrust) — unrust — A pure rust based (webgl 2.0 / native) game engine
+* [Godot](https://godotengine.org/)
+  * [godot-rust/godot-rust](https://github.com/godot-rust/godot-rust) [[gdnative](https://crates.io/crates/gdnative)] - Rust bindings to the Godot game engine [<https://api.travis-ci.com/godot-rust/godot-rust.svg?branch=master>](https://travis-ci.com/github/godot-rust/godot-rust)
```

Comment:
`<https://api.travis-ci.com/godot-rust/godot-rust.svg?branch=master>` is just a link. It should either be `<img src="https://api.travis-ci.com/godot-rust/godot-rust.svg?branch=master" />` or `![build badge](https://api.travis-ci.com/godot-rust/godot-rust.svg?branch=master)`

Reply:
oops :Ddone

Reply:
Closer, but not quite. It's now `<img="` rather than `<img src="`

Reply:
facepalms myself.... done and I hope for the last time